### ISSUE TITLE
Add support for “Bot” owner type

### ISSUE THREAD

**Issue by kirelagin** [onrock-eng/github-webhooks#18]
My GitHub App is listening for `InstallationRepositoriesEvent` and it also sometimes creates repositories itself and then receives an event about the repository it just created. Events for repositories created by the app have the following `sender`:

```
  "sender": {
    "login": "<application-slug>[bot]",
    <...>
    "type": "Bot",
    <...>
  }
```

Sender is being parsed as `HookUser`, whose `type` field is parsed as `OwnerType`, which currently supports only two values (`User` or `Organization`), so it fails with:

```
Error in $.sender.type: Unknown owner type: Bot
```

**Comment by donkeybonks** [comment]
Hey, thanks for reporting this.

Is there official documentation on this behaviour somewhere we can use?

**Comment by kirelagin** [comment]
No, I have never seen any documentation mentioning any `sender` type other than `User` or any comprehensive schema definition, I’m sorry.

**Comment by donkeybonks** [comment]
Gents, this is released `0.11.0`.
